libnavigation-core/api/current.txt

```diff
@@ -676,11 +676,13 @@ package com.mapbox.navigation.core.routealternatives {
   }
 
   public final class AlternativeRouteMetadata {
+    method public int getAlternativeId();
```

Comment:
Did we miss these docs in some other PR?

Reply:
Yes, @VysotskiVadim forgot to update these files in #6329.